Resources/Prototypes/Entities/Structures/barricades.yml

```diff
@@ -38,8 +38,8 @@
       - !type:SpawnEntitiesBehavior
         spawn:
           MaterialWoodPlank1:
-            min: 3
-            max: 3
+            min: 2 # The construction graphs that use this all override this value on all trees so it's just a fallback
```

Comment:
This isn't fallback; this is what spawns when you _destroy_ the barricade, i.e. hitting it with weapons. 

Using prying to deconstruct the barricade should be considered the more "careful" option compared to just smashing it. Because of that, make smashing the barricade leave 1 plank (because one breaks) while prying via the graph gives back the full 2 planks.

Reply:
Thanks for the assistance, still learning how this all works :)